packages/react-sdk/src/helpers/caching/contentTypes/attachment.ts

```diff
@@ -9,41 +9,110 @@ import {
   RemoteAttachmentCodec,
 } from "@xmtp/content-type-remote-attachment";
 import { ContentTypeId } from "@xmtp/xmtp-js";
+import { z } from "zod";
```

Comment:
thoughts on adding an alias for this import just for easier readability?

Reply:
i'm not sure i understand. can you elaborate?